clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -117,6 +121,77 @@ def start_clipper(
+                                    application, input_type, default_value,
+                                    slo)
+                            except ClipperException as e:
+                                print("required parameters not found")
+                                raise e
+                            except KeyError, f:
+                                print("required parameters not found")
+                                raise ClipperException(KeyError)
+                    if ("models" in config):
+                        for model in config["models"]:
+                            try:
+                                config_image = config["models"][model]["image"]
+                                input_type = config["models"][model][
+                                    "input_type"]
+                                version = config["models"][model]["version"]
+                                labels = None
+                                if ("labels" in config["models"][model]):
+                                    labels = config["models"][model]["labels"]
+                                num_replicas = 1
+                                if ("num_replicas" in config["models"][model]):
+                                    num_replicas = config["models"][model][
+                                        "num_replicas"]
+                                batch_size = -1
+                                if ("batch_size" in config["models"][model]):
+                                    batch_size = config["models"][model][
+                                        "batch_size"]
+                                self.deploy_model(model, version, input_type,
+                                                  config_image, labels,
+                                                  num_replicas, batch_size)
+                            except ClipperException as e:
+                                print("required parameters not found")
+                                raise e
+                            except KeyError, f:
+                                print("required parameters not found")
+                                raise ClipperException(KeyError)
+                    if ("links" in config):
+                        for link in config["links"]:
+                            try:
+                                self.link_model_to_app(
+                                    config["links"][link]["app_name"],
+                                    config["links"][link]["model_name"])
+                            except ClipperException as e:
+                                print("required parameters not found")
+                                raise e
+                            except KeyError, f:
+                                print("required parameters not found")
+                                raise ClipperException(KeyError)
+
+        parse_configuration(config_file)
```

Comment:
Check for the existence of config_file before calling parse_configuration here:

```py
if config_file is not None:
    parse_configuration(config_file)
```

Reply:
Done